lib/matplotlib/axes/_base.py

```diff
@@ -112,14 +112,14 @@ def _process_plot_format(fmt):
                 raise ValueError(
                     'Illegal format string "%s"; two marker symbols' % fmt)
             marker = c
-        elif c in mcolors.colorConverter.colors:
+        elif c in mcolors.get_named_colors_mapping():
             if color is not None:
                 raise ValueError(
                     'Illegal format string "%s"; two color symbols' % fmt)
             color = c
         elif c == 'C' and i < len(chars) - 1:
             color_cycle_number = int(chars[i + 1])
-            color = mcolors.colorConverter._get_nth_color(color_cycle_number)
+            color = mcolors.to_rgba("C{}".format(color_cycle_number))
```

Comment:
This seems like a wash either way.  The `elif` statement above is encoding the details of the formatting.  Using `_get_nth_color` only buries half the body.

On the other hand, it is our policy that we can use mpl private API anywhere in the mpl code base (that is the _point_ of the private APIs).